Kinvey-Xamarin/Offline/SQLTemplates.cs

```diff
@@ -82,14 +82,6 @@ public OfflineMetaData(string id, JObject customHeaders)
 
 			public OfflineMetaData(){}
```

Comment:
@tejasranade Not sure, though I suspect not.  I decided to just address the unnecessary exception being thrown for OfflineVersion in this PR, but I will submit another PR soon where I plan to clean up the codebase of any unused code left over from 1.x, since I think we're close enough to release that we can delete unused legacy code.